<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-95GUOUG" failures="1" tests="5" name="Personal Loan" time="0.098" errors="0" timestamp="2023-04-13T18:16:50 IST">
  <testcase classname="firsttestngpackage.FirstTestNGZFile" name="demo" time="0.015">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at firsttestngpackage.FirstTestNGZFile.demo(FirstTestNGZFile.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
  <testcase classname="firsttestngpackage.Day2" name="demo" time="0.003"/>
  <testcase classname="firsttestngpackage.Day3" name="loginAPIHomeLoan" time="0.003"/>
  <testcase classname="firsttestngpackage.Day3" name="mobileLoginHomeLoan" time="0.002"/>
  <testcase classname="firsttestngpackage.Day3" name="webLoginHomeLoan" time="0.003"/>
  <testcase classname="firsttestngpackage.FirstTestNGZFile" name="SecondTest" time="0.0">
    <ignored/>
  </testcase> <!-- SecondTest -->
</testsuite> <!-- Personal Loan -->
